src/graphql/fetchAllRequestIdentifiers.ts

```diff
@@ -79,19 +79,28 @@ export async function fetchAllRequestIdentifiers(
   }
 
   do {
-    // eslint-disable-next-line no-await-in-loop
-    const response = await sombra!
-      .post<{
-        /** Decrypted identifiers */
-        identifiers: RequestIdentifier[];
-      }>('v1/request-identifiers', {
-        json: {
-          first: PAGE_SIZE,
-          offset,
-          requestId,
-        },
-      })
-      .json();
+    let response: unknown;
+    try {
+      // eslint-disable-next-line no-await-in-loop
+      response = await sombra!
+        .post<{
+          /** Decrypted identifiers */
+          identifiers: RequestIdentifier[];
+        }>('v1/request-identifiers', {
+          json: {
+            first: PAGE_SIZE,
+            offset,
+            requestId,
+          },
+        })
+        .json();
+    } catch (err) {
+      throw new Error(
+        `Failed to fetch request identifiers: ${
+          err?.response?.body || err?.message
```

Comment:
Is this too dangerous?
```suggestion
          err?.response?.body || err?.message || JSON.stringify(err)
```
(I recently hit a case where an error had neither a response.body nor a message is why I ask.)

Reply:
id rather not do that - by default it shows error.message which right now is just showing a 400 error with no message, this code makes it so that it shows the inner error message, which got hides by default in the response field. JSON.stringify seems unnecessary here